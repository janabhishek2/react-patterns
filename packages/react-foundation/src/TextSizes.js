export default {
    xs: 'xs',
    sm: 'sm',
    base: 'base',
    lg: 'lg',
    xl: 'xl'
};