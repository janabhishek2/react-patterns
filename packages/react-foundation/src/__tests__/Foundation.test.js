import { SPACING } from "../Spacing";
import TextSizes from "../TextSizes";

describe('Snapshot test foundations', () => { 
    test('contains the correct Spacing snapshot', () => { 
        expect(SPACING).toMatchSnapshot();
    });

    test('contains the correct TextSizes snapshot', () => {
        expect(TextSizes).toMatchSnapshot();
    })
});
