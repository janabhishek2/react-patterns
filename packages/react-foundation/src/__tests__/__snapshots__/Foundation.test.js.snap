// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test foundations contains the correct Spacing snapshot 1`] = `
{
  "lg": "lg",
  "md": "md",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
  "xxl": "xxl",
  "xxs": "xxs",
  "xxxl": "xxxl",
  "xxxs": "xxxs",
}
`;

exports[`Snapshot test foundations contains the correct TextSizes snapshot 1`] = `
{
  "base": "base",
  "lg": "lg",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
}
`;
